eng/pipelines/dailytests.yml

```diff
@@ -32,6 +32,9 @@ parameters:
 - name: isOfficialBuild
   type: boolean
   default: false
+- name: testMachineImageName
+  type: string
+  default: Windows-11-Enterprise-23H2-WinBuildDevFiles
```

Comment:
Can someone who has better understanding of the machine pools review this: @NuGet/nuget-client

Reply:
FWIW, I onboarded NuGet onto DartLab, but I don't know anything about their VM images. The default always worked for us for the Apex & E2E machines, and by using their defaults, it also meant we didn't need to maintain the image name over the months & years when new versions of Windows were released, when compliance required changes to image names, etc.

Setting a specific image name here might be a fine temporary mitigation, but I believe we should contact the DartLab team to understand why there's no longer enough disk space, and to see if they're going to change the default image, so that we don't have to maintain the VM image name in our yaml long term.

Reply:
Thanks @zivkan 
Let's start with that. 

Maybe we can try to find the contact info...start with: https://github.com/NuGet/Client.Engineering/blob/main/docs/CI/build-pipeline.md#dartlab, and file this with them.

Reply:
@SueSu01 Can you try contacting?
If you prefer, someone from the team can do it instead.

Reply:
> @SueSu01 Can you try contacting? If you prefer, someone from the team can do it instead.

@nkolev92 I sent an email to the ImageFactory team. Thanks.